src/Uno.Toolkit.UI/Behaviors/TabBarSelectorBehaviorState.Uwp.cs

```diff
@@ -77,7 +77,7 @@ private void OnScrolViewerViewChanged(object? sender, ScrollViewerViewChangedEve
 			// -------------
 			// 1   2   3   4    
 			var offset =
-#if WINDOWS_UWP
+#if WINDOWS || WINDOWS_UWP
```

Comment:
File name ending with `.Uwp.cs` looks confusing, unless I'm missing something.

Reply:
yeah I'll change it to .Windows.cs